src/__tests__/__snapshots__/MessageImage.test.js.snap

```diff
@@ -37,14 +37,7 @@ exports[`should render <MessageImage /> and compare with snapshot 1`] = `
         onResponderTerminate={[Function]}
         onResponderTerminationRequest={[Function]}
         onStartShouldSetResponder={[Function]}
-        style={
-          Array [
-            Object {
-              "backgroundColor": "transparent",
-            },
-            undefined,
-          ]
-        }
+        style={null}
```

Comment:
Any idea what caused this snapshot to change? Have you verified that MessageImage still works as expected without `backgroundColor`?

Reply:
Not tested yet